ahnlich/ai/src/tests/aiproxy_test.rs

```diff
@@ -606,7 +627,8 @@ async fn test_ai_proxy_binary_store_actions() {
     let message = AIServerQuery::from_queries(&[
         AIQuery::CreateStore {
             store: store_name.clone(),
-            model: AIModel::Llama3,
+            query_model: AIModel::Llama3,
+            index_model: AIModel::DALLE3,
```

Comment:
The query model and index model have a mismatch in embedding size, why is the createstore still having an Ok